.github/workflows/vscode-extension.yml

```diff
@@ -45,20 +45,35 @@ jobs:
         with:
           node-version: 16.16.x
 
+      - name: Install Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: "1.23.4"
+
       - name: Attempt to rebuild
         run: npm rebuild | true
       
       - name: Install Dependencies
         run: npm ci
 
       - name: Compile packages
-        run: npm run build-all
+        run: |
+          echo "$(pwd)" >> $GITHUB_PATH
+          npm run build-all
+
+      - name: Make taq binary globally visible
+        run: |
+          echo $(which taq) >> $GITHUB_PATH
+
+      - name: update run-tests.sh to make it executable
+        run: chmod +x ./taqueria-vscode-extension/bin/run-tests.sh
       
       - name: Compile TS
         run: npm run compile:test --workspace ${{ env.GIT_WORKSPACE }}
       
       - name: Run Unit Tests on Linux
-        run: xvfb-run -a npm test --workspace ${{ env.GIT_WORKSPACE }}
+        run: |
```

Comment:
Doesn't need to be piped since there is only one command

Reply:
Fixed.